src/fsharp/FSharp.Core/option.fsi

```diff
@@ -23,6 +28,11 @@ module Option =
     ///
     /// <param name="option">The input option.</param>
     ///
+    /// <remarks>
+    /// <code>
+    /// None |> Option.isNone = true
+    /// Some 42 |> Option.isNone = false</code></remarks>
```

Comment:
Please put each code snippet in an `<example>` section not `<remarks>`.  If it's not being shown in output of fsdocs let's modify fsdocs, though I think it is, see https://github.com/fsprojects/FSharp.Formatting/blob/420e4a4eb3152fedd5ff647a68a73a5ea1fd79cd/src/FSharp.Formatting.ApiDocs/GenerateHtml.fs#L168

Also put `</code></example>` on a new line, it's hard to read where it is

Reply:
We can also reposition where the examples are shown by modifying fsdocs

Reply:
Ack.

Reply:
When </code></example> was on a newline I remember there being an ugly blank line at the end of the code block. I will double check. If there is, I'll still eliminate the newline here but will file an issue with fsdocs to eliminate the ugly blank line. Okay?

Reply:
The empty line looks like this:

![image](https://user-images.githubusercontent.com/6829565/107893579-d75f2a80-6ee0-11eb-93f1-40f16c8ce8ca.png)

It still repros so I'll file an issue with fsdocs.

Reply:
Hmmm yes these are fsdocs problems, thanks.